diff --git a/CMake/Dependencies.cmake b/CMake/Dependencies.cmake
index 2c83a46d7..c4a2d9d56 100644
--- a/CMake/Dependencies.cmake
+++ b/CMake/Dependencies.cmake
@@ -178,33 +178,6 @@ if(OGRE_BUILD_DEPENDENCIES)
         --build ${PROJECT_BINARY_DIR}/assimp-6.0.2 ${BUILD_COMMAND_OPTS})
     endif()
 
-    message(STATUS "Building Bullet")
-    file(DOWNLOAD
-        https://github.com/bulletphysics/bullet3/archive/refs/tags/3.25.tar.gz
-        ${PROJECT_BINARY_DIR}/3.25.tar.gz)
-    execute_process(COMMAND ${CMAKE_COMMAND}
-        -E tar xf 3.25.tar.gz WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
-    execute_process(COMMAND ${BUILD_COMMAND_COMMON}
-        -DBUILD_SHARED_LIBS=OFF
-        -DINSTALL_LIBS=ON
-        -DCMAKE_POSITION_INDEPENDENT_CODE=ON
-        -DUSE_MSVC_RUNTIME_LIBRARY_DLL=ON
-        -DBUILD_PYBULLET=OFF
-        -DUSE_DOUBLE_PRECISION=OFF
-        -DBUILD_CPU_DEMOS=OFF
-        -DBUILD_BULLET2_DEMOS=OFF
-        -DBUILD_EXTRAS=OFF
-        -DBUILD_EGL=OFF
-        -DBUILD_ENET=OFF
-        -DBUILD_UNIT_TESTS=OFF
-        -DCMAKE_RELWITHDEBINFO_POSTFIX= # fixes FindBullet on MSVC
-        -DBUILD_CLSOCKET=OFF
-        -DCMAKE_POLICY_VERSION_MINIMUM=3.5
-        ${PROJECT_BINARY_DIR}/bullet3-3.25
-        WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/bullet3-3.25)
-    execute_process(COMMAND ${CMAKE_COMMAND}
-        --build ${PROJECT_BINARY_DIR}/bullet3-3.25 ${BUILD_COMMAND_OPTS})
-    set(BULLET_ROOT ${OGREDEPS_PATH})
 endif()
 
 #######################################################################
@@ -309,8 +282,4 @@ find_package(assimp QUIET)
 macro_log_feature(assimp_FOUND "Assimp" "Needed for the AssimpLoader Plugin" "https://www.assimp.org/")
 
-# Bullet
-find_package(Bullet QUIET)
-macro_log_feature(BULLET_FOUND "Bullet" "Bullet physics" "https://pybullet.org")
-
 if(assimp_FOUND)
   # workaround horribly broken assimp cmake, fixed with assimp 5.1
   add_library(fix::assimp INTERFACE IMPORTED)
